src/main/scala/org/codeoverflow/chatoverflow/requirement/service/file/FileConnector.scala

```diff
@@ -0,0 +1,45 @@
+package org.codeoverflow.chatoverflow.requirement.service.file
+
+import org.codeoverflow.chatoverflow.WithLogger
+import org.codeoverflow.chatoverflow.connector.Connector
+import org.codeoverflow.chatoverflow.connector.actor.FileSystemActor
+import org.codeoverflow.chatoverflow.connector.actor.FileSystemActor._
+
+class FileConnector(override val sourceIdentifier: String) extends Connector(sourceIdentifier) with WithLogger {
+  override protected var requiredCredentialKeys: List[String] = List()
+  override protected var optionalCredentialKeys: List[String] = List()
+  private val fileActor = createActor[FileSystemActor]()
+
+  def getFile(pathInResources: String): Option[String] = {
+    if (fileActor.??[Some[String]](5) {LoadFile(pathInResources)}.isDefined) {
```

Comment:
Could be centralized for better performance

Reply:
Should be done.